src/test/ui/associated-consts/issue-24949-assoc-const-static-recursion-trait-default.stderr

```diff
@@ -36,6 +37,7 @@ note: ...which requires caching mir of `FooDefault::BAR` for CTFE...
 LL |     const BAR: u32 = DEFAULT_REF_BAR;
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    = note: ...which requires normalizing `DEFAULT_REF_BAR`...
+   = note: ...which requires normalizing `DEFAULT_REF_BAR`...
```

Comment:
What can we do about this duplication? We can't just remove the query description of normalize_erasing... queries.